docs/how-to-connect/desktop-sdks/qt-app.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Qt Desktop App
+description: An Qt desktop app example for Casdoor
+keywords: [qt, sdk]
+---
+
+A [Qt desktop app example](https://github.com/casdoor/casdoor-cpp-qt-example) for Casdoor.
+
+# How to run example
+
+## Prerequisites
+
+[Qt6 sdk](https://www.qt.io/download)
+
+[OpenSSL toolkit](https://www.openssl.org/source/)
+
+## Initialization
+
+You need to init requires 7 parameters, which are all string type:
+
+| Name         | Description                                                                                             | File                  |
+| ------------ | ------------------------------------------------------------------------------------------------------- | --------------------- |
+| endpoint       | Your Casdoor server host/domain                                                                         | `mainwindow.h` |
+| client_id     | The Client ID of your Casdoor application                                                               | `mainwindow.h` |
+| client_secret | The Client Secret of your Casdoor application                                                           | `mainwindow.h` |
+| certificate      | The public key for the Casdoor application's cert                                                                    | `mainwindow.h` |
+| org_name      | The name of your Casdoor organization                                                                    | `mainwindow.h` |
+| app_name      | The name of your Casdoor application                                                                    | `mainwindow.h` |
+| redirect_url  | The path of the callback URL for your Casdoor application, will be `http://localhost:8080/callback` if not provided | `mainwindow.h` |
+
+If you don't set these parameters, this project will use the [Casdoor localhost demo](http://localhost:8080) as the defult Casdoor server.
```

Comment:
Do you mean if  parameter `endpoint`  is not initialized?  These parameters are vaguely expressed and may mislead the reader.

Reply:
> Do you mean if parameter `endpoint` is not initialized? These parameters are vaguely expressed and may mislead the reader.

Yes, that's it. I will modify the expression here later.